dashboard/test/models/section_test.rb

```diff
@@ -84,19 +84,15 @@ class SectionTest < ActiveSupport::TestCase
   end
 
   test 'name is required' do
-    assert_does_not_create(Section) do
-      section = Section.new user: @teacher
-      refute section.valid?
-      assert_equal ['Name is required'], section.errors.full_messages
-    end
+    section = Section.new user: @teacher
```

Comment:
Because `build :section` would, by default, provide a name. Though on reflection, I like `section = build :section, name: nil`. That work for you?